Sources/HCILEAddDeviceToWhiteList.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  HCILEAddDeviceToWhiteList.swift
+//  Bluetooth
+//
+//  Created by Carlos Duclos on 6/14/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import Foundation
+
+public extension HCILowEnergyCommand {
+    
+    /// LE Add Device To White List Command
+    ///
+    /// The command is used to add a single device to the White List stored in the Controller.
+    ///
+    /// This command can be used at any time except when:
+    /// * any advertising filter policy uses the White List and advertising is enabled.
+    /// * the scanning filter policy uses the White List and scanning is enabled.
+    /// * the initiator filter policy uses the White List and a create connection command is outstanding.
+    ///
+    /// When a Controller cannot add a device to the White List because there is no space available,
+    /// it shall return the error code Memory Capacity Exceeded (0x07).
+    ///
+    /// Address is ignored when Address Type is set to 0xFF.
+    public struct HCILEAddDeviceToWhiteList: HCICommandParameter { //HCI_LE_Add_Device_To_White_List
```

Comment:
@carlos21 Rename to `HCILEAddDeviceToWhiteList` from `HCILowEnergyCommand.HCILEAddDeviceToWhiteList`. Remove extension.